packages/dom/test/utils.test.js

```diff
@@ -0,0 +1,13 @@
+import { styleSheetFromNode } from '../src/utils';
+
+describe('utils', () => {
```

Comment:
```suggestion
describe('styleSheetFromNode', () => {
```

convention is `describe` block contains the function/class name